packages/components/tsconfig.json

```diff
@@ -1,25 +1,26 @@
 {
   "compilerOptions": {
-    "outDir": "dist/",
-    "module": "commonjs",
-    "target": "es5",
-    "lib": ["es5", "es6", "dom"],
-    "sourceMap": true,
     "allowJs": false,
-    "jsx": "react",
-    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
     "baseUrl": "src",
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
```

Comment:
I'd remove this unless we're using decorators.

As far as I can tell, decorators is in [stage 2 of TC39](https://tc39.es/proposal-decorators/), which means it's not yet accepted as part of standard JS.

Having said that, if we think we're going to use it, then it's fine.